env.sh

```diff
@@ -74,6 +75,17 @@ install_cmssw() {
   fi
 }
 
+install_cmssw_addons() {
+  #install CMSSW updates not in official releases
+  #TODO: Consider to add more sophisticated version matching
+  local CMSSW_VER=$1
+  if [ "X$CMSSW_VER" = "XCMSSW_14_0_6_patch1" ]; then
```

Comment:
If there are known issues that prevent us from doing it, I think we'll want to switch to the latest 14_0_X version for the new production. Is `cms-tau-pog:CMSSW_14_0_X_tau-pog_BoostedDeepTau` compatible `CMSSW_14_0_17`?
A better way to implement it is to pass argument `apply_addons` to `install_cmssw`, like this one can have a more transparent control on it.

Reply:
I think it should be compatible with all 14_0_X releases => 14_0_6_patch1, but I will check it particularly against 14_0_17. If it is not the case I will rebase the branch to 14_0_17.

To be sure: is your proposal to change call `install_cmssw_addons()` within `install_cmssw` with an initial CMSSW version check in the former function (with an additional `if` statment)?

Reply:
I confirm that `cms-tau-pog:CMSSW_14_0_X_tau-pog_BoostedDeepTau` is compatible.